tests/tendermint/conftest.py

```diff
@@ -7,6 +7,22 @@ def b():
     return BigchainDB()
 
 
+@pytest.fixture
+def alice():
+    from bigchaindb.common.crypto import generate_key_pair
+    return generate_key_pair()
+
+
+@pytest.fixture
+def alice_privkey(alice):
+    return alice.private_key
+
+
+@pytest.fixture
+def alice_pubkey(alice):
+    return alice.public_key
+
```

Comment:
The above fixtures exist in the [`tests/conftest.py`](https://github.com/bigchaindb/bigchaindb/blob/master/tests/conftest.py#L202) too. Any reason they have been re-defined here?

Reply:
Will clean up.